pages/api/v1/standings/preseason.ts

```diff
@@ -0,0 +1,93 @@
+import { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
What makes this api endpoint different and can we put it in the main standings api endpoint instead?

Reply:
There is no team_records for pre-season, I've built a view in mysql to simulate something similar using the schedule table. 

I can try to incorporate it into the regular standings api, but the way it's read and parsed is different in a few spots, even though the output is very similar. I'm not sure exactly how to blend the two into the same endpoint, this seemed to be the easiest solution.